macos/ReactTestApp/AppDelegate.swift

```diff
@@ -150,9 +150,19 @@ final class AppDelegate: NSObject, NSApplicationDelegate {
 
         switch component.presentationStyle {
         case "modal":
-            viewController.view.frame = NSRect(size: WindowSize.modalSize)
+            let modalFrame = NSRect(size: WindowSize.modalSize)
+            viewController.view.frame = modalFrame
             if let rootView = viewController.view as? RCTRootView {
-                rootView.minimumSize = WindowSize.modalSize
+                var token: NSObjectProtocol?
```

Comment:
Should it be a `let` and assign on same line?   Or, actually, if its not used don't need to assign token at all?

Reply:
It gets captured so we can unsubscribe on line 163.

Reply:
Got it, somehow missed that